src/commands/cluster-flushslot.json

```
@@ -0,0 +1,46 @@
+{
+    "FLUSHSLOT": {
+        "summary": "Remove all keys from the target slot.",
+        "complexity": "O(N) where N is the number of keys in the target slot",
+        "group": "cluster",
+        "since": "8.1.0",
+        "arity": -3,
+        "container": "CLUSTER",
+        "function": "clusterCommand",
+        "command_flags": [
+            "WRITE"
+        ],
+        "acl_categories": [
+            "KEYSPACE",
+            "DANGEROUS"
+        ],
```

I think we should block this command on async loading. also, `ADMIN` is a more appropriate flag, which gets `DANGEROUS` automatically.
```suggestion
        "command_flags": [
            "WRITE",
            "NO_ASYNC_LOADING",
            "ADMIN"
        ],
        "acl_categories": [
            "KEYSPACE"
        ],
```